### DISCUSSION TITLE
Not working... I havent even updated

### DISCUSSION THREAD

**Discussion by BluePenguin-666** [3kh0/ext-remover#51] [Help JOIN TITANIUM NETWORK TO SHARE PLEASE] [answered]
This thing was working for like a week but now whenever I click the bookmarklet it doesn't work. I haven't updated

**Comment by CheesyPoofs3** [comment]
same but with every bookmark

**Comment by 3kh0** [comment, marked as answer]
@BluePenguin-666 Can you send a screenshot of the URLBlocklist policy?

**Reply by BluePenguin-666** [reply]
Were do i find that?

**Reply by 3kh0** [reply]
go to chrome://policy

**Reply by BluePenguin-666** [reply]
What do you want a screenshot of?

**Reply by 3kh0** [reply]
ctrl+f and put in URLBlocklist, find the policy, and click view more, and screenshot that

**Reply by BluePenguin-666** [reply]
![image](https://user-images.githubusercontent.com/70232660/196461739-b78db251-7291-4469-a182-e77a3f2c6fc7.png)

**Reply by BluePenguin-666** [reply]
ohhh, so the school blocked javascript:// wich means bookmarklets dont work, well it was fun while it lasted.
